db/migrate/20181128140935_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,56 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    table = :users
+    change_table(table) do |t|
+      ## Required
+      t.string :provider, :null => false, :default => "email" unless column_exists?(table, :provider)
+      t.string :uid, :null => false, :default => "" unless column_exists?(table, :uid)
+
+      ## Database authenticatable
+      t.string :encrypted_password, :null => false, :default => "" unless column_exists?(table, :encrypted_password)
+
+      ## Recoverable
```

Comment:
We already have this columns, there is no need to add this lines in this migration

Reply:
Ok